packages/react-vapor/src/components/pagination/PaginationUtils.tsx

```diff
@@ -1,7 +1,25 @@
+import {createSelector} from 'reselect';
+import * as _ from 'underscore';
+import {IReactVaporState} from '../../ReactVapor';
+import {IPaginationState} from '../navigation/pagination';
+
 const getPaginationPagesNumbersId = (id: string) => `${id}_PagesNumber`;
 const getPaginationPerPageId = (id: string) => `${id}_PerPage`;
 
 export const PaginationUtils = {
     getPaginationPagesNumbersId,
     getPaginationPerPageId,
 };
+
+const getPaginationState = (state: IReactVaporState, {id}: {id: string}) =>
+    _.findWhere(state.paginationComposite, {id: id});
+
+const getPaginationPageNumber = createSelector(
+    getPaginationState,
+    (paginationState: IPaginationState): number => paginationState?.pageNb ?? 0
+);
+
+export const PaginationSelectors = {
```

Comment:
Rename the file to `PaginationSelectors`

Reply:
Not renamed but I created a new files and moved the PaginationSelectors